readme.md

```diff
@@ -84,6 +84,91 @@ Please see the following reference implementations for additional examples and t
   - [Libauth](https://github.com/bitauth/libauth) – An ultra-lightweight, zero-dependency JavaScript library for Bitcoin Cash. [Pull Request #139](https://github.com/bitauth/libauth/pull/139).
   - [Bitauth IDE](https://github.com/bitauth/bitauth-ide) – An online IDE for bitcoin (cash) contracts. [Pull Request #101](https://github.com/bitauth/bitauth-ide/pull/101).
 
+## Activation Costs
+
+We define activation cost as any cost that would arise ex-ante, in anticipation of the activation by affected stakeholders, to ensure that their services will continue to function without interruption once the activation block has been mined.
+In case of this proposal, activation cost is contained to nodes and libraries that implement the Script VM, and will amount to:
+
+- Some fixed amount of node developer man-hours, in order to release updated node software that will be able to correctly calculate the algorithmic limit.
+- Some fixed amount of work by stakeholders running node software, in order to update the software in anticipation of algorithm activation.
+- Some fixed amount of effort by others involved in reviewing and testing the new node software version.
+- Some fixed amount of effort by library maintainers in order to update their VM implementations to correctly execute arithmetic operations that would be failing pre-upgrade.
+
+This is the same kind of cost that any upgrade to the Script VM system has had or will have.
+Nothing is required of stakeholders who are neither running nodes nor involved in node development or smart contract development.
```

Comment:
If I understand correctly, this is only true for smart contract app developers, especially those needing to make web wallet implementations (basically all of them), if their apps and wallets do not violate the new targeted limits. I don't know how strongly we can say that. If we can say it strictly due to design constraints relative to old constraints, that would be great. If not, then it should be mentioned that there is a possibility of some apps being broken.

Reply:
I guess you are calling that "smart contract development".

So this group should be added to the fixed amount of effort group.

Reply:
I don't understand what you're saying here. How could existing apps/wallets violate new targeted limits when the limits will be more generous than current ones?

Reply:
That's exactly the point. You are saying here that is true, but that is not explicitly stated anywhere. And I also was not confident from reading that literally every limit is strictly 100% same or looser. That's a very strong statement to make and needs to be made if the claim of "nothing is required of ..." is to be stated.

Reply:
"And I also was not confident from reading that literally every limit is strictly 100% same or looser."

this is the big-int chip, I think this comment belongs in VM limits chip

Reply:
the only way to break something with bigint is if some contract somehow uses int64 overflow as a switch, and there's a whole section on that: https://github.com/bitjson/bch-bigint/blob/4ff3d21146db452a67235a4c3b4e3ce010d26945/readme.md#risks-to-existing-smart-contracts